src/types.ts

```diff
@@ -54,6 +54,7 @@ export enum ScannerProperty {
   SonarScannerSonarCloudURL = 'sonar.scanner.sonarcloudUrl',
   SonarScannerJavaExePath = 'sonar.scanner.javaExePath',
   SonarScannerWasEngineCacheHit = 'sonar.scanner.wasEngineCacheHit',
+  SonarScannerInternalSqVersion = 'sonar.scanner.internal.sqVersion',
```

Comment:
for backwards compatability we would also need to check for `sonar.scanner.versionSimulation`

Reply:
I don't see that the current version of NPM scanner supports `versionSimulation`, am I missing something here?